### PR TITLE
Create a simple autocomplete field using HTML ONLY

### PR DESCRIPTION
**Pull request by akiyamaSM**
it would be something like

```
Route::get('/info', function(){
    $languages = ['PHP', 'JAVA', 'JAVASCRIPT', 'C#', 'C++'];
    return view('welcome', compact('languages'));
});
```

in the view 

```
<form action="/" method="POST">
    {{ Form::autoComplete("firstName", null, ["class" => "form-control"], ['myId' => $languages]) }}
    <input type="submit" />
</form>
```

### CODE REVIEW

**Comment by mlantz** [comment]
Please ensure your code passes all checks. I'm also concerned that Safari doesnt support the datalist at all. http://caniuse.com/#feat=datalist

**Comment by akiyamaSM** [comment]
I've fixed the styling @mlantz, and yes, it's not supported in SAFARI

File: src/FormBuilder.php [hunk 1]
```diff
@@ -1211,4 +1211,70 @@ public function __call($method, $parameters)
...
+        return $this->inputAutoComplete('text', $name, $value, $options, $complete, $list);
+    }
+
+    /**
+     * Prepare the datalist based on the array and the id.
+     * @param array $list
+     * @param $id
+     * @return String
+     */
+    protected function prepareDateList(array $list, $id)
```
Comment (hebbet) [inline src/FormBuilder.php]: maybe you should change the name to prepareDat**a**List

**Comment by tshafer** [comment]
Please provide a test and resubmit to master branch.

**Pull request closed without merging**